CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ Remember to align the itemized text with the first line of an item within a list
 PLEASE REMEMBER TO CHANGE THE '..main' WITH AN ACTUAL TAG in GITHUB LINK.
 -->
 
-## jax 0.3.14 (Unreleased)
+## jax 0.3.14 (June 21, 2022)
```

Comment:
Add the unreleased versions too, so people can easily add new CHANGELOG items

Reply:
Done,